riva/clients/nmt/riva_nmt_streaming_s2s_client.cc

```diff
@@ -71,6 +72,9 @@ DEFINE_bool(
     "Whether to use SSL credentials or not. If ssl_cert is specified, "
     "this is assumed to be true");
 DEFINE_string(metadata, "", "Comma separated key-value pair(s) of metadata to be sent to server");
+DEFINE_string(tts_prosody_rate, "", "Speech rate for TTS output");
+DEFINE_string(tts_prosody_pitch, "", "Speech pitch for TTS output");
+DEFINE_string(tts_prosody_volume, "", "Speech volume for TTS output");
```

Comment:
one question, if user passes just one of these parameters to the clients and doesn't pass the other two, does it work fine? in this case, we will be sending empty values of pitch and volume to the server.

Reply:
Yes it will work fine.

Reply:
would setting the default value of these fields to "default" instead of empty be better?

Reply:
it is handled on server side, resolving